src/cloudai/schema/test_template/jax_toolbox/slurm_command_gen_strategy.py

```diff
@@ -262,11 +263,11 @@ def set_xla_flags(test_name: str, profile_enabled: bool):
 
         # Combine both parts into the run script content
         run_script_content = profile_content + perf_content
-        run_script_path = os.path.join(cmd_args["output_path"], "run.sh")
+        run_script_path = Path(cmd_args["output_path"]) / "run.sh"
```

Comment:
_[minor]_ can be `run_script_path = output_path / "run.sh"` for readability.

Reply:
Are you expecting code like this?
```
        output_path = Path(cmd_args["output_path"])
        run_script_path = output_path / "run.sh"
```

Reply:
Sorry, diff was confusing, it collapsed in-between lines like this:
![image](https://github.com/user-attachments/assets/f672f75d-f1e1-4ad8-8320-6005bd8963ed)
so I decided that `output_path` is in the same scope still. Please ignore.